corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -346,6 +353,8 @@ hqDefine("app_manager/js/details/case_claim", function () {
                     allowBlankValue: searchProperties[i].allow_blank_value,
                     exclude: searchProperties[i].exclude,
                     required: searchProperties[i].required,
+                    validationXPath: validation ? validation.xpath : '',
+                    validationMessage: validation ? validation.message[lang] : '',
```

Comment:
Just pattern matching, it's odd that validationMessage references the language, but hint and label, both translatable strings, don't. Would it be doable to handle all of these the same way?

Reply:
They do reference `lang`, it's just higher up in the function - I wouldn't object to moving those in-line if that's what you're thinking.
https://github.com/dimagi/commcare-hq/blob/fd574f67ddb3061685de878cd33c02b59c8dbf69/corehq/apps/app_manager/static/app_manager/js/details/case_claim.js#L321-L322

Reply:
aha, got it. I do tend to update this file in a very mechanical, copy-paste kind of way each time we add a new case search feature, but I don't think it's a big deal either way.